test/org.jkiss.dbeaver.ext.greenplum.test/OSGI-INF/l10n/bundle_ro_RO.properties

```diff
@@ -0,0 +1,21 @@
+#Generated by ResourceBundle Editor (http://essiembre.github.io/eclipse-rbe/)
+
+datasource.greenplum.description = Surs\u0103 de date Greenplum
```

Comment:
I think the file pass is incorrect here. We do not have localizations in our tests folder.

This is correct one: plugins/org.jkiss.dbeaver.ext.greenplum/OSGI-INF/l10n/bundle.properties

Reply:
Solved in [dc41b1b](https://github.com/v613/dbeaver/commit/dc41b1b8db11db37af504a75957881ef7145b9a8)